dev-db/pgcli/pgcli-3.0.0.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit distutils-r1
+
+DESCRIPTION="CLI for Postgres with auto-completion and syntax highlighting"
+HOMEPAGE="https://www.pgcli.com https://github.com/dbcli/pgcli"
+SRC_URI="https://github.com/dbcli/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	>=dev-python/prompt_toolkit-3.0.0[${PYTHON_USEDEP}]
+	<dev-python/prompt_toolkit-4.0.0[${PYTHON_USEDEP}]
+	>=dev-python/psycopg-2.8.0[${PYTHON_USEDEP}]
+	>=dev-python/sqlparse-0.3.0[${PYTHON_USEDEP}]
+	<dev-python/sqlparse-0.4.0[${PYTHON_USEDEP}]
+	>=dev-python/cli_helpers-1.2.0[${PYTHON_USEDEP}]
+	>=dev-python/click-4.1[${PYTHON_USEDEP}]
+	>=dev-python/configobj-5.0.6[${PYTHON_USEDEP}]
+	>=dev-python/humanize-0.5.1[${PYTHON_USEDEP}]
+	>=dev-python/pgspecial-1.11.8[${PYTHON_USEDEP}]
+	>=dev-python/pygments-2.0[${PYTHON_USEDEP}]
+	>=dev-python/setproctitle-1.1.9[${PYTHON_USEDEP}]
+"
+DEPEND="${RDEPEND}"
+
+BDEPEND="
+	test? (
+		dev-db/postgresql
+		dev-python/mock[${PYTHON_USEDEP}]
+	)
+"
+
+# there is a flaky test, so no tests for now
+RESTRICT="test"
```

Comment:
Is there a way to remove/disable that flaky test, and run the others?

Reply:
Sure, how can I do that ?

Reply:
I ran `pgcli` tests a dozen of times, the flaky seems gone and all is green. I will update the PR but I would still like to know how to ignore specific tests in ebuild :)